packages/components/src/tooltip/types.ts

```diff
@@ -58,6 +58,12 @@ export type TooltipProps = {
 	 * The text shown in the tooltip when anchor element is focused or hovered.
 	 */
 	text?: string;
+	/**
+	 * The custom class name for the tooltip.
+	 * The default class for tooltip is `components-tooltip` and always will be added.
+	 * The custom class name will be appended to the default class.
```

Comment:
Having a default class and appending a custom one to it is pretty regular behavior, so probably the description could be shortened to something like:

```suggestion
	 * Custom class name for the tooltip, in addition to the default `components-tooltip`.
```